contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,118 @@
+
+
+### They don’t clash
+
+Creating a startup full of startups sounds like a recipe for people duplicating work or stepping on toes, right? Wrong. If you combine a small team structure with a transparent culture where everything is as public as possible and, ideally, written down, then everyone should have more than enough context of what others are working on and why. 
+
+At PostHog, we have a list of everything we do [and who owns it](/handbook/engineering/feature-ownership). This helps avoid clashes by clearly establishing feature ownership. If a product or project doesn't have a clear owner, our [fuzzy ownership process](/handbook/company/fuzzy-ownership) encourages people to create a PR and find a solution.
+
+
+Giving clear ownership over products and projects to individuals or individual teams also lessens the risk of overlap.
+
+We’ve got a small number of product managers who help small teams prioritize, create and update dashboards, conduct competitor analysis, and speak to customers. But we also expect product engineers to be true experts of the product they’re building and understand our customers. Product managers also ensure cross-team alignment, an important task in a completely remote, asynchronous company.
+
+So what about stepping on toes? Well, we [actively encourage that](https://posthog.com/handbook/values). Everyone at PostHog should be raising pull requests in GitHub or proposing changes to stuff that doesn't have anything to do with your small team. Why? Because it shows they really care and they really understand the product.
+
+
+### They have their own mission
+
+Regardless of size or scope, each team has its own mission that feeds into our overall company mission of equipping every developer to build successful products.
+
+On the [marketing team](https://posthog.com/teams/marketing), the mission is to make PostHog a place for product engineers to learn and collaborate. The [infrastructure team’s mission](https://posthog.com/teams/infrastructure) is to make deploying, scaling, and managing PostHog easy, fast, and reliable. But a charter is more than a mission. Small team charters also need to include:
+
+* Long-term goals
+* A description of what the team does
+* A target customer
+* Details of who’s on the team
+* Key metrics
+
+All of this information is in our [handbook](https://posthog.com/handbook) and updated when changes are made and confirmed each quarter. This keeps everyone on the same page and makes it easy for anyone within the company to quickly find out what other teams are working on and why.
+
+### They're not just engineers
```

Comment:
I feel like we can lose this one. Open to counter argument, but it doesn't feel essential atm.

Reply:
Yeah, I'm not especially wedded to it so happy to cut. Will add it in as a line elsewhere to make the point that small teams aren't just engineering.